### DISCUSSION TITLE
Add support for transfer-learned collective variables (arXiv:2404.03777)

### DISCUSSION THREAD

**Discussion by svandenhaute** [ACEsuit/mace#372] [Ideas]
We recently developed a simple transfer learning approach based on MACE. The node features of a trained model are fed into a simple classifier readout, which computes a per-atom (and total) logit vector for a given atomic geometry. This vector is used to predict 'phases' (or any other discrete label) to atomic geometries while exploiting the fact that the network features have been trained to be 'informative' for the atomic interactions. These phase predictions can then be used to bias the system from one phase to another, and compute relative stabilities. The preprint is [here](https://arxiv.org/abs/2404.03777).

Our code builds off of the main MACE repository, and basically augments `atomic_data` with a `phase` attribute which holds the target label to learn, `modules.loss` with a mixed energy / forces / cross-entropy loss, and a calculator which allows for bias potentials to be applied on the output logits (see preprint for details). Would it be worthwhile for us to prepare a PR which merges this functionality in the main code base?
